controller.go

```diff
@@ -198,13 +198,13 @@ func (c *controller) NetworkByID(id string) Network {
 	return nil
 }
 
-func (c *controller) sandboxAdd(key string) (sandbox.Sandbox, error) {
+func (c *controller) sandboxAdd(key string, create bool) (sandbox.Sandbox, error) {
```

Comment:
`create` is misleading. can you please name it more appropriately such as `createNS` ?